server/jetstream_cluster_4_test.go

```diff
@@ -1369,13 +1371,11 @@ func TestJetStreamClusterStreamOrphanMsgsAndReplicasDrifting(t *testing.T) {
 	// Clustered file based with discard old policy and no limits.
 	t.Run("R3F_DO_NOLIMIT", func(t *testing.T) {
 		params := &testParams{
-			restartAny:       false,
-			ldmRestart:       true,
-			rolloutRestart:   true,
-			restarts:         3,
-			checkHealthz:     true,
-			reconnectRoutes:  true,
```

Comment:
These were to trigger some specific failure conditions and as it is this version of the test is at 99% reliability at the moment, but to track down the current issue I think sounds good to change these to being the same for now.